csplit/src/csplit.yml

```diff
@@ -0,0 +1,64 @@
+name: csplit
+version: "0.1.0"
+author: Mats Kindahl <mats.kindahl@gmail.com>
+about: |-
+  Split FILE by each PATTERN into a series of files 'xx00', 'xx01',
+  ... 
+
+  A file name has to be provided, but if it is '-', input will read
+  from standard input.
+
+  The input will be split up to, but not including, the line that
+  matches PATTERN.
+
+  The number of bytes in each file will be displayed on standard
+  output with one line for each file.
+after_help: |-
+  PATTERN can be any of:
+    INTEGER            copy lines up to line number
+    /REGEXP/[OFFSET]   copy lines up to line matching REGEXP
+    %REGEXP%[OFFSET]   skip lines up to line matching REGEXP
+    {INTEGER}          repeat preceeding pattern INTEGER times
+    {*}                repeat preceeding pattern indefinitely
+
+  If an OFFSET is given it should be an integer, either positive or
+  negative. An offset without sign is assumed to be positive.
```

Comment:
I'm not sure if this fits best here or as `long_help` for `PATTERN`. It was ok in both for me.

Try both and give a feedback, please.

Reply:
`long_help` looked better.